sendinblue.php

```diff
@@ -32,16 +32,19 @@
     include_once (_PS_CLASS_DIR_ . '/../classes/Customer.php');
```

Comment:
dirname(FILE)  not working  for prestashop version 1.4.x, in module list page .
and to make it working for 1.4.x we are using _PS_CLASS_DIR_ for other version we are using as you asked.